examples/tools/scene_viewer/scene_viewer_plugin.rs

```diff
@@ -198,35 +198,13 @@ fn keyboard_animation_control(
     }
 }
 
-const SCALE_STEP: f32 = 0.1;
-
 fn update_lights(
     key_input: Res<Input<KeyCode>>,
     time: Res<Time>,
     mut query: Query<(&mut Transform, &mut DirectionalLight)>,
     mut animate_directional_light: Local<bool>,
 ) {
-    let mut projection_adjustment = Vec3::ONE;
-    if key_input.just_pressed(KeyCode::Key5) {
-        projection_adjustment.x -= SCALE_STEP;
-    } else if key_input.just_pressed(KeyCode::Key6) {
-        projection_adjustment.x += SCALE_STEP;
-    } else if key_input.just_pressed(KeyCode::Key7) {
-        projection_adjustment.y -= SCALE_STEP;
-    } else if key_input.just_pressed(KeyCode::Key8) {
-        projection_adjustment.y += SCALE_STEP;
-    } else if key_input.just_pressed(KeyCode::Key9) {
-        projection_adjustment.z -= SCALE_STEP;
-    } else if key_input.just_pressed(KeyCode::Key0) {
-        projection_adjustment.z += SCALE_STEP;
-    }
```

Comment:
The instructions also need to be removed from the `impl fmt::Display for SceneHandle`.

Reply:
Done.